doc/themes/the-things-stack/layouts/404.html

```diff
@@ -1,10 +1,8 @@
-{{ define "main" }}
-{{ if eq .Page.File.Ext "html" }}
-{{ .Content }}
-{{ else }}
-<h1 class="title is-size-2">{{ .Title }}</h1>
-
-<div class="content">{{ .Content }}</div>
-{{ end }}
-{{ partial "prev-next-in-section.html" . }}
+{{ define "main"}}
+  <div class="container">
+    <div class="section has-text-centered">
+      <h1 class="title is-size-3 has-vertical-space">Whoops! This page doesn't exist.</h1>
+      <a href="{{ "/" | relURL }}">Go Home</a>
```

Comment:
I cracked up a little on the page telling you to "Go Home" :joy: 

```suggestion
      <a href="{{ "/" | relURL }}">Go to homepage</a>
```